cmd/ignite/run/create_test.go

```diff
@@ -221,7 +322,23 @@ func TestNewCreateOptions(t *testing.T) {
 
 	for _, rt := range tests {
 		t.Run(rt.name, func(t *testing.T) {
-			_, err := rt.createFlag.NewCreateOptions([]string{})
+			// Setup storage backend.
+			dir, err := ioutil.TempDir("", "ignite")
+			if err != nil {
+				t.Fatalf("failed to create storage for ignite: %v", err)
+			}
+			defer os.RemoveAll(dir)
+
+			storage := cache.NewCache(
```

Comment:
This should use Ignite's configured providers to have consistency between the tests and built binary, can be done in a separate PR.